examples/hmm.py

```diff
@@ -286,6 +351,8 @@ def main(args):
     parser.add_argument("-n", "--num-steps", default=50, type=int)
     parser.add_argument("-b", "--batch-size", default=8, type=int)
     parser.add_argument("-d", "--hidden-dim", default=16, type=int)
+    parser.add_argument("-nn", "--nn-dim", default=48, type=int)
```

Comment:
please fix or remove `capacity` bit

Reply:
Hmm I believe this should still be a correct computation of capacity, i.e. the total number of parameters. Can you explain in person?